spec/fluent.asdl

```diff
@@ -25,12 +25,16 @@ module Fluent
 {
     res = Resource(entry* body, comment? comment, span? span)
 
-    entry = Message(iden id, pat? value, attr* attributes, tag* tags, comment? comment)
+    entry = BaseMessage(base_message)
           | Section(symb name, comment? comment)
           | Comment(comment)
           | Junk(str content)
           attributes (annot* annotations, span? span)
 
+    base_message = Message(pat? value, attr* attributes)
+                 | PrivateMessage(pat value, tag* tags)
```

Comment:
This makes it a requirement that `PrivateMessages` have a non-null `value`. Should we enforce this for now or keep it consistent with public `Messages`?

Reply:
I'm ok enforcing it.